src/main/resources/db/changelog/changes/05-insert-users.yaml

```
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-users
+      author: student
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column:
+                  name: email
+                  value: "admin@example.com"
+              - column:
+                  name: password
+                  value: "$2a$10$r5Ih0Fdb/DATMcG41JKdiuBsrhslSH4qZ9wuDcj9qRpZwtUzDHLOa"
```

please add 
```suggestion
                  value: "$2a$10$r5Ih0Fdb/DATMcG41JKdiuBsrhslSH4qZ9wuDcj9qRpZwtUzDHLOa" # 123456
```
please add passwords in plaintext as comment